sessions/backend-basics/routes/README.md

```diff
@@ -0,0 +1,44 @@
+# Backend Basics: Routes
+
+In this challenge, you will build a server which responds depending on the url route opened on the browser.
+
+## Task
+
+Build a server that sends a different response depending on the url route:
```

Comment:
Just an idea... In the draft of this session we wrote a note:

> Idea (maybe): Perhaps we could have a conceptually unifying project separated into little projects or concepts for the backend part.
Something along the lines of the inventory of a store, such as "My little (Pet/Candle/Soap/Hardware ...) store", which could (!) include a personal interest, and a vaguely sensible application.

I quite like this idea. 

In a challenge for the next session, [Backend API](https://github.com/neuefische/web-exercises/pull/230/files#diff-25b3d5cef55c6de2197536b4ce738a99aa6fcd60938a616e50b086bee9736c5d) we introduced API routes for  products (fish).

To follow the idea, we could start here and let the students create routes, that matcht this category

```
- if the requested url is `/api/products/1`, the response should be `Shrimp`;
- if the requested url is `/api/products/2`, the response should be `Anemonefish`;
```

Reply:
Great idea, I'm going to adapt this!